### PR TITLE
[global-hooks] Fix `global.storageClass` => `global.modules.storageClass` conversion

### PR DESCRIPTION
**Pull request by Ranger-X**
## Description

Fixes unpredictable behavior when moving a parameter from `global.storageClass` to `global.modules.storageClass` (see #9859).
Also cleaning up `global` v2 specs from obsolete/deprecated properties.

## Why do we need it, and what problem does it solve?

Fix bug with `global` conversions introduced in PR #9859.

## What is the expected result?

Correct definition of `storageClass`-es for Deckhouse's modules.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: global-hooks
type: fix
summary: Fix bug with `global.storageClass` => `global.modules.storageClass`.
impact: 
impact_level: low
```

### CODE REVIEW

File: global-hooks/openapi/conversions/v2.yaml
Comment (Ranger-X) [inline global-hooks/openapi/conversions/v2.yaml:1]: Should I save this file on Git for future coders or remove it and conversions folder?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/eks'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Ranger-X. I've started the workflow for label 'e2e/run/aws'...

**Review by yalosev — APPROVED** [review]

**Pull request merged**